### ISSUE TITLE
Namespace conflict with "SMS API" extension

### ISSUE THREAD

**Issue by MegaphoneJon** [JohnFF/SMS-Inbox#7]
This extension defines an API `Sms.send`, which is the same name as that provided by the `SMS API` extension.

I can't find any reference to the API being called within the extension except via unit tests.  Is it OK to remove?

**Comment by JohnFF** [comment]
I don't think it can be, as we need the ability to send from the inbox, which probably isn't covered by UTs. But if it's defined now, it probably wasn't when I first wrote it. Could you propose a refactor to use this new SMS.send?

**Comment by MegaphoneJon** [comment]
Hi @JohnFF - the SMS API extension defined this API about 5 years ago, which looks like it predates yours by a year - but I don't think the "who was first" argument is compelling.  However, I think your Sms.send is unused - here's the part of your code that sends SMSes:

```php
    // Try sending the message.
    try {
      list($sent, $activityId, $countSuccess) = CRM_Activity_BAO_Activity::sendSMS($recipientDetailsArray, $activityParams, $smsParams, $contactIdArray, NULL);
```
